### DISCUSSION TITLE
bootstrap

### DISCUSSION THREAD

**Discussion by Jesim0** [twbs/bootstrap#34706] [Q&A]
good day... I am done learning HTML and CSS can I proceed to learning bootstrap without learning javascript? thanks

**Comment by X7md** [comment]
You can, but I think it's will be not good idea.

I think if you have basic understanding of JavaScript, then learn bootstrap will be better. this will be make you think better about how use bootstrap with javascript.

for example:

loop through javascript array to add item inside list then give a list good basic style using bootstrap classes and so on...
